CreeDictionary/DatabaseManager/management/commands/xmlimport.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+
+from django.core.management import BaseCommand, call_command
+
+
+class Command(BaseCommand):
+    help = """Erase all rows from all definition-related database tables.
+
+    This is useful to run before importing a new dictionary.
+    """
```

Comment:
Oh man I forgot this was in this PR

this is a copy paste error from me

```suggestion
    help = """Import a dictionary .xml file into db search tables.

    At a high level, this will (1) analyze each wordform to get a lemma, then
    (2) generate all wordforms of the lemmas. During this process, definitions
    are associated with wordforms, and are also stemmed to allow searching.
    """
```